compiler+runtime/test/cpp/jank/read/lex.cpp

```diff
@@ -374,7 +374,61 @@ namespace jank::read::lex
         }));
       }
     }
-
+    TEST_CASE("Ratio")
+    {
+      SUBCASE("Successes")
```

Comment:
Please split this subcase up into named subcases, like all of our other tests. This allows you to avoid the `p`, `p1`, `p2` problem, too.

Reply:
splitted

Reply:
split the test for the "success" cases as well.